src/android/StatusBar.java

```diff
@@ -64,6 +64,9 @@ public void run() {
 
                 // Read 'StatusBarStyle' from config.xml, default is 'lightcontent'.
                 setStatusBarStyle(preferences.getString("StatusBarStyle", "lightcontent"));
+
+                // Read 'StatusBarOverlaysWebView' from config.xml, default is false.
+                setStatusBarTransparent(preferences.getBoolean("StatusBarOverlaysWebView", false));
```

Comment:
While the default is broken, I'd rather fix the bug in the way that it corresponds with the docs, i.e., true.